src/cli/start.rs

```diff
@@ -183,17 +183,20 @@ pub async fn init(
 	// Start the kvs server
 	dbs::init(dbs).await?;
 	// Start the node agent
-	// This is equivalent to run_maintenance in native/wasm drivers
-	let nd = node::init(ct.clone());
-	let lq = node::live_query_change_feed(ct.clone());
+	let tasks = surrealdb::tasks::start_tasks(
+		&config::CF.get().unwrap().engine.unwrap(),
+		ct.clone(),
+		DB.get().unwrap(),
+	)
+	.await;
 	// Start the web server
 	net::init(ct).await?;
 	// Wait for the node agent to stop
-	if let Err(e) = nd.await {
+	if let Err(e) = tasks.nd.await {
 		error!("Node agent failed while running: {}", e);
 		return Err(Error::NodeAgent);
 	}
-	if let Err(e) = lq.await {
+	if let Err(e) = tasks.lq.await {
 		error!("Live query change feed failed while running: {}", e);
 		return Err(Error::NodeAgent);
 	}
```

Comment:
I would do a common Err handling here and hide the actual tasks inside, so when we add more we don't need to add more Err handling here.